processor.js

```diff
@@ -15,6 +15,6 @@ module.exports = function processor(src, options) {
   loaderPromises[options.path || 'auto'] = loaderPromise
 
   return loaderPromise
-    .then((plugins) => postcss(plugins).process(src, { from: false }))
+    .then((plugins) => postcss(plugins).process(src, { from: options.babel.filename, map: false }))
```

Comment:
This will break if `options.babel` is undefined. Can you add a guard?

Reply:
Sure, I've added a fallback. Just to let you known: I couldn't test it in my setup anymore, since I ended up not relying on this plugin.